plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/rest/TrinoRestCatalog.java

```diff
@@ -370,13 +391,32 @@ public void setTablePrincipal(ConnectorSession session, SchemaTableName schemaTa
     @Override
     public void createView(ConnectorSession session, SchemaTableName schemaViewName, ConnectorViewDefinition definition, boolean replace)
     {
-        throw new TrinoException(NOT_SUPPORTED, "createView is not supported for Iceberg REST catalog");
+        ViewBuilder viewBuilder = restSessionCatalog.buildView(convert(session), toIdentifier(schemaViewName));
```

Comment:
I'd move this down to be part of the larger chain starting with `viewBuilder.withSchema`